### PR TITLE
Drop glide, use dep.

### PR DESCRIPTION
**Pull request by dajohi**

### CODE REVIEW

File: run_tests.sh [hunk 1]
```diff
@@ -40,8 +41,8 @@ docker run --rm -it -v $(pwd):/src decred/$DOCKER_IMAGE_TAG /bin/bash -c "\
   rsync -ra --filter=':- .gitignore'  \
   /src/ /go/src/github.com/decred/$REPO/ && \
   cd github.com/decred/$REPO/ && \
-  glide install && \
-  go install \$(glide novendor) && \
+  dep ensure && \
+  go install ./... && \
```
Comment (chappjc) [inline run_tests.sh]: didn't you want to use ${DIRS} here too?

File: run_tests.sh [hunk 2]
```diff
@@ -40,8 +40,7 @@ docker run --rm -it -v $(pwd):/src decred/$DOCKER_IMAGE_TAG /bin/bash -c "\
   rsync -ra --filter=':- .gitignore'  \
   /src/ /go/src/github.com/decred/$REPO/ && \
   cd github.com/decred/$REPO/ && \
-  glide install && \
-  go install \$(glide novendor) && \
+  dep ensure \
```
Comment (teknico) [inline run_tests.sh]: Missing the `&&` before the ending backslash.

Comment (dajohi) [reply run_tests.sh]: thanks

**Comment by teknico** [comment]
And now Travis is failing because of incompatible cross-constraints in manifest, which will hopefully be solved by decred/dcrd#877.

**Comment by teknico** [comment]
And now it fails because of the outstanding problem in `gometalineter`, see #60.

**Comment by jrick** [comment]
i would just close this pr, no point in updating this. the repo will be nuked after everything has been moved out of it

**Pull request closed without merging**